.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -24,5 +24,6 @@
 - [ ] My code follows the code style of this project.
 - [ ] My change requires a change to the documentation.
 - [ ] I have updated the documentation accordingly.
-- [ ] I have added tests to cover my changes.
+- [ ] I have added unit tests to cover my changes.
+- [ ] I have added/updated functional tests in `mint`. (If yes, add `mint` PR # here: )
```

Comment:
Why do we need this? and does minio repository has functional test?

Reply:
> Why do we need this?

We need this because there are PRs fixing issues in Minio server, but corresponding functional tests are not getting added to Mint. Hopefully this will remind developers/reviewers to add/ask to add tests for Mint. 

Currently we are creating issues to add tests in Mint after a fix is sent to Minio server, which is an additional task and can be brought into our workflow. 

> does minio repository have functional test?

No. I am not sure what are you implying here.